cirq/ops/common_gates.py

```diff
@@ -122,23 +122,32 @@ def controlled(self,
                    control_qid_shape: Optional[Tuple[int, ...]] = None
                   ) -> raw_types.Gate:
         """
-        Constructs CXPowGate from controlled XPowGate when applicable.
-
-        This method is a specialized controlled method for XPowGate. It
-        overrides the default behavior of returning a ControlledGate by
-        transforming the underlying controlled gate to a CXPowGate and
-        removing the last specified control qubit (which acts first
-        semantically).  If this is a gate with multiple control qubits, it will
-        now be a ControlledGate with one less control.
-
-        This behavior only occurs when the last control qubit is a default-type
-        control qubit. A default-type control qubit is one with shape of 2 (not
-        a generic qudit) and where the control is satisfied by the qubit being
-        ON, as opposed to OFF. Note also that this only transforms into a
-        CXPowGate (or controlled version of that gate) if the global shift on
-        the XPowGate is 0, otherwise it produces a normal ControlledGate.
-
-        (Note that a CXPowGate is, by definition, a controlled-XPowGate.)
+        Returns a `CXPowGate` or a controlled `CXPowGate` when possible.
+
+        The `controlled` method of the `Gate` class, of which this class is a
+        child, returns a `ControlledGate`. This method overrides this behavior
+        to return a `CXPowGate` or a `ControlledGate` of a `CXPowGate, when
```

Comment:
```suggestion
        to return a `CXPowGate` or a `ControlledGate` of a `XPowGate, when
```

Reply:
Same as above. Also need to add ` before "," here and below.